src/index.js

```diff
@@ -158,11 +167,7 @@ class Popover extends Component {
         target={(selectionRef && selectionRef.current) || document}
         onMouseDown={() => this.setState({ mousePressed: true })}
       />,
-      position == null || !isOpen ? null : (
-        <div key="popup" className={className} style={style} ref={measureRef}>
-          {children}
-        </div>
-      )
```

Comment:
so all you should need to do change to the desired outcome is this:

```
const isOpen = this.props.isOpen != null ? props.isOpen : this.state.isOpen
```

That should be all really

Reply:
i thought of keeping that logic part  separately.Will make the change and thanks for your patience!

Reply:
@juliankrispel  `props.isOpen` would be`undefined right?

Reply:
@sunilhari actually even better - you can check if it is a boolean `typeof this.props.isOpen === 'boolean'` - that's even stricter!

Reply:
Respect brother :)